lib/engine/game/g_18_mex/step/buy_sell_par_shares.rb

```diff
@@ -36,6 +45,17 @@ def can_buy_any?(entity)
           def attempt_ndm_action_on_unavailable?(bundle)
             bundle.corporation == @game.ndm && @game.phase.status.include?('ndm_unavailable')
           end
+
+          # Need to allow buying 5% shares in NdM even if at cert limit as these shares are not
+          # counted towards cert limit (but they still count for 60% corporation limit).
+          def room_to_gain?(entity, bundle, exchange)
+            return true if bundle.corporation == @game.ndm &&
+                           @game.num_certs(entity) == @game.cert_limit &&
+                           bundle.percent == 5 &&
+                           !exchange
+
+            @game.num_certs(entity) < @game.cert_limit
```

Comment:
I personally think it's clearer to do the normal check (`@game.num_certs(entity) < @game.cert_limit`) first and then deal with the remaining exception, but it's not that important.

Reply:
Agreed, have updated now.